ui/src/app/shared/components/tags-input/tags-input.tsx

```diff
@@ -0,0 +1,113 @@
+import * as classNames from 'classnames';
+import * as React from 'react';
+
+import {Autocomplete, AutocompleteApi, AutocompleteOption} from 'argo-ui';
+
+export interface TagsInputProps {
+    tags: string[];
+    autocomplete?: (AutocompleteOption | string)[];
+    onChange?: (tags: string[]) => any;
+    placeholder?: string;
+}
+
+require('./tags-input.scss');
+
+export class TagsInput extends React.Component<TagsInputProps, {tags: string[]; input: string; focused: boolean}> {
```

Comment:
@alexmt should we be pushing this into the shared Argo UI code? does that still exist?